README.md

```diff
@@ -23,88 +22,101 @@ applications yield the processor and transported using RDMA-like operations, res
 minimal jitter during collection. LDMS has been run on 10,000 cores collecting
 over 100,000 metric values per second with less than 0.2% overhead.
 
-## Scalable Storage System (SOS)
+# Building the OVIS / LDMS source code
 
-SOS is a high-performance, indexed, object-oriented database designed to efficiently
-manage structured data on persistent media. More information can be found at
-the SOS project on GitHub website <https://github.com/ovis-hpc/SOS>.
+## Obtaining the source code
 
-## Baler
+You may obtain the source code by obtaining an official release tarball, or by
+cloning the ovis-hpc/ovis [Git](http://git-scm.com/) repository at github.
 
-Baler is an aggregation of log message exploration and analysis tools. _balerd_ is
-the tool that tokenizes log messages using user-specified dictionaries.
-The log messages are then grouped together according to their token sequences.
-Each group is represented by a Baler pattern -- a token sequence.
+### Release tarballs
 
-Baler stores the log message patterns, the raw log messages, and other infomation in
-its database. _bquery_ is a tool to query Baler database for the Baler patterns,
-raw log messages, and message statistics by hosts and/or time.
+Official Release tarballs are available from the GitHub releases page:
 
-Baler also comes with an association mining tool -- _bassoc_ -- that can be used
-to discover sequence of occurrence patterns of log messages and to perform causal analysis.
+  https://github.com/ovis-hpc/ovis/releases
 
-Baler has been moved out of the OVIS tree. Please check it out at <https://github.com/opengridcomputing/baler>
+The tarball is avialble in the "Assets" section of each release. Be sure to
+download the tarball that has a name of the form "ovis-ldms-X.X.X.tar.gz".
 
-# Obtaining OVIS source code
+The links that are named "Source code (zip)" and "Source code (tar.gz)" are
+automatic GitHub links that we are unable to remove. Yhey will be missing the
+configure script, because they are raw source from git repository and
+not the official release tarball distribution.
 
-You may clone OVIS source (and its submodules) from the official [Git](http://git-scm.com/) repository:
+### Cloning the git repository
 
-```sh
-git clone https://github.com/ovis-hpc/ovis.git -b OVIS-4
-# If you are interested in storing LDMS data in the SOS storage
-# please perform the following step.
-git clone https://github.com/ovis-hpc/sos.git -b SOS-4
-```
+To clone the source code, go to https://github/com/ovis-hpc/ovis, and click
+one the "Code" button. Or use the following command:
 
-# Dependencies
+```git clone https://github.com/ovis-hpc/ovis.git -b OVIS-4```
 
-* autoconf (>=2.63), automake, libtool
-* glib2
+## Build Dependencies
+
+* autoconf (>=2.63)
+* automake
+* libtool
+* make
+* bison
+* flex
+* gettext development package
+* libevent developmetn package
```

Comment:
*development

Reply:
@morrone actually, we don't need libevent anymore. I think the comment in `configure.ac` stating that we do was a copy-paste error.

Reply:
OK.